lib/stat_tracker.rb

```diff
@@ -375,4 +391,19 @@ def game_info_by_team(team_id)
       game_team['team_id'] == team_id
     end
   end
-end
\ No newline at end of file
+
+  def result_totals_by_team(team_id)
+    result = {}
+    result[:total]  = game_info_by_team(team_id).length
+    result[:wins]   = game_info_by_team(team_id).select do |game|
+                        game['result'] == "WIN"
+                      end.length
+    result[:ties]   = game_info_by_team(team_id).select do |game|
+                        game['result'] == "TIE"
+                      end.length
+    result[:losses] = game_info_by_team(team_id).select do |game|
+                        game['result'] == "LOSS"
+                      end.length
```

Comment:
Great job @smcunning! My only comment would be on this section here

I feel like there is an opportunity for refactoring this since you're using almost the exact same select method a couple different times. Is there a way to break that out into a helper method?

Reply:
I agree with @taphill, it does look a helper method would work well here. Other than that this looks great to me.

Reply:
Cool, refactored this with Taylor's suggestion!